SIMExplPhaseField.C

```diff
@@ -0,0 +1,94 @@
+// $Id$
+//==============================================================================
+//!
+//! \file SIMExplPhaseField.C
+//!
+//! \date May 27 2016
+//!
+//! \author Knut Morten Okstad / SINTEF
+//!
+//! \brief Solution driver representing an explicit phase-field.
+//!
+//==============================================================================
+
+#include "SIMExplPhaseField.h"
+#include "SIMoutput.h"
+#include "TimeStep.h"
+#include "DataExporter.h"
+#include "Functions.h"
+#include "Utilities.h"
+#include "IFEM.h"
+#include "tinyxml.h"
+
+
+SIMExplPhaseField::SIMExplPhaseField (SIMoutput* gridOwner)
+{
+  myHeading = "Explcit phase field";
+  myOwner   = gridOwner;
+  phaseFunc = nullptr;
+  myStep    = 0;
+}
+
+
+void SIMExplPhaseField::registerFields (DataExporter& exporter)
+{
+  exporter.registerField("c","phase field",DataExporter::SIM,
+                         DataExporter::PRIMARY);
+  exporter.setFieldValue("c",this,&phaseField);
+}
+
+
+bool SIMExplPhaseField::init (const TimeStep&)
+{
+  this->registerField("phasefield",phaseField);
+  return true;
+}
+
+
+bool SIMExplPhaseField::parse (const TiXmlElement* elem)
+{
+  const char* value = utl::getValue(elem,"phasefield");
+  if (!value) return false;
```

Comment:
this doesn't seem right. we are passed all subtags of <simulation>, not all will match phasefield and returning false here will halt parsing.

Reply:
Indeed wrong. Should here forward to parent-class method for all other tags. Don't know when that vanished but fixed now. Probably something went wrong during a rebase at one point.